quantum/nodes/local-simulator/local-simulator.html

```diff
@@ -9,16 +9,16 @@
     inputs: 1,
     outputs: 1,
     icon: "font-awesome/fa-microchip",
-    paletteLabel: "simulator",
+    paletteLabel: "local simulator",
     label: function () {
-      return this.name || "simulator";
+      return this.name || "Local Simulator";
```

Comment:
```suggestion
      return this.name || "local simulator";
```
For consistency, this should be lowercase.